### PR TITLE
run_all_tests.sh is no longer available, run tests separately

### PR DESCRIPTION
**Pull request by frznlogic**

### CODE REVIEW

**Review by tobsan — CHANGES_REQUESTED** [review]
I'm not sure we want to run all examples automatically, but apart from that this looks good.

File: Vagrantfile [hunk 1]
```diff
@@ -70,8 +70,12 @@ Vagrant.configure(2) do |config|
                 path: "cookbook/build/clang-code-analysis.sh"
 
             config.vm.provision "shell", inline: <<-SHELL
-                   cd softwarecontainer
-                   sudo su -c ./run-all-tests.sh
+                   cd ~/softwarecontainer/build
+                   sudo ./run-tests.sh
```
Comment (thenor) [inline Vagrantfile]: Shouldn't this be "run-tests.py"?

Comment (frznlogic) [reply Vagrantfile]: You are correct. Fixed.

**Comment by frznlogic** [comment]
@tobsan would you like me to remove the whole section instead? It's currently inside a if ENV['BUILD_AND_TEST'] then so it's not run by default.

**Review by thenor — APPROVED** [review]

**Comment by tobsan** [comment]
@frznlogic yes, automatically building and testing is done by the Jenkinsfile nowadays anyway.

**Comment by frznlogic** [comment]
@tobsan updated documentation and removed the section completely.

File: README.md [hunk 2]
```diff
@@ -159,21 +159,19 @@ information.
...
 # Testing
 There are currently three levels of tests: unit tests, component tests, and
-service tests. To run all tests after building, run `run-all-tests.sh`
-in the project root.
+service tests.
 
 The unit tests written in gtest/gmock are available in */unit-test/*. The tests
-are run using `run-tests.sh` from the build directory. Note that you have
+are run using `run-tests.py` from the build directory. Note that you have
 to run them as root, since the actual container creation is not stubbed of
```
Comment (thenor) [inline README.md]: Please take the opportunity to fix the "of" typo. It should be "off", or not included at all.

Comment (frznlogic) [reply README.md]: done.

**Review by DunderRoffe — APPROVED** [review]

**Review by tobsan — APPROVED** [review]

**Pull request merged**